### ISSUE TITLE
Bacteria data download

### ISSUE THREAD

**Issue by SierraStreamsInstitute-Lab** [thosmos/riverdb#42]
When we download data table 2, we get everything except bacteria data.

**Comment by thosmos** [comment]
I'm not sure I understand.  When I run data table 2, I see TotalColiform and EColi columns
<img width="483" alt="Screen Shot 2023-02-15 at 2 17 28 AM" src="https://user-images.githubusercontent.com/621286/218999768-461386e3-e9a7-46fa-86ca-bda23d862097.png">

**Comment by SierraStreamsInstitute-Lab** [comment]
Yes, I see that too, it's when I download CSV for all years that it's missing.

**Comment by SierraStreamsInstitute-Lab** [comment]
Oh, it got weirder! 
![CSV download](https://user-images.githubusercontent.com/114112659/219122261-22082115-9488-4d6a-98f6-2a53725c7016.png)

**Comment by thosmos** [comment]
That doesn’t look very good

**Comment by SierraStreamsInstitute-Lab** [comment]
Just checking on this here, I'm still getting the same blank csv file. 
I also wanted to check the format of bulk uploads; what I have from when Alex was working on this is this:
[RiverDB data upload template.xlsx](https://github.com/thosmos/riverdb/files/10926301/RiverDB.data.upload.template.xlsx)
But it doesn't have some necessary info like device name. The download template used to though:
[RiverDB data download format.xlsx](https://github.com/thosmos/riverdb/files/10926304/RiverDB.data.download.format.xlsx)

**Comment by thosmos** [comment]
The All Years button has been fixed.  As for the device column, normally you would have a device column per parameter with the device ID for those values, and that ID should match with an existing device ID in your devices table.  It looks like the report download only shows the default device type for each parameter, but device IDs are missing entirely.  These were never imported manually before.  Did you ever enter these when adding data manually?

**Comment by SierraStreamsInstitute-Lab** [comment]
I believe we did enter them when adding data manually, and most data that you pulled from RiverDB has a device associated with it.
